### PR TITLE
PR for Feature  profile details

### PR DESCRIPTION
**Pull request by jeremyrist**

### CODE REVIEW

**Comment by jeffworley** [comment]
Nice work and I def dig the use of different shapes.  You truly don't see shape.  lol :+1:

**Comment by s4dmw** [comment]
looks good. liking those circles and ovals...  :+1:

**Comment by ShayHall** [comment]
Nice skitch fest and good clean opening on those boxes.  I like Iron- :octocat:  Make sure the left column ends up shorter.

File: src/index.html [hunk 1]
```diff
@@ -7,16 +7,45 @@
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
+        <link rel="stylesheet" href="octicons/octicons.css">
```
Comment (kellymurray) [inline src/index.html:10]: You can link the Octicon CDN here - easier than uploading all the font files.

File: src/index.html [hunk 2]
```diff
@@ -7,16 +7,45 @@
...
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
         <!-- Place favicon.ico in the root directory -->
 
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
+
         <script src="js/vendor/modernizr-2.8.3.min.js"></script>
     </head>
     <body>
       <div class="container">
```
Comment (kellymurray) [inline src/index.html:20]: Diiiiiiiiivs! Let's make it semantic!

**Pull request merged**